addons/web_editor/static/lib/odoo-editor/test/utils.js

```diff
@@ -272,7 +277,10 @@ export function customErrorMessage(assertLocation, value, expected) {
     return `[${assertLocation}]\nactual  : '${value}'\nexpected: '${expected}'\n\nStackTrace `;
 }
 
-export async function testEditor(Editor = OdooEditor, spec, options = {}) {
+export async function testEditor(Editor = OdooEditor, spec, options = {}, isMobile = false) {
```

Comment:
The current version of the PR does _not_ run the mobile tests because `isMobile` has been moved to `options` at call site but not here.

Reply:
That's quite a blunder, sorry about that.
This time I checked that all tests are run in their mobile version. 
FYI there are 272 tests affected, for an overall increase in testing time of around 3 seconds for me (from 23s to 26s).